app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/AddressController.scala

```diff
@@ -86,7 +86,7 @@ trait AddressController[J <: JourneyStatus] {
             updateSession(sessionStore, request)(_.copy(addressLookupResult = None)).map {
               case Left(e) =>
                 logger.warn(s"Could not clear addressLookupResult", e)
-                errorHandler.errorResult(request.userType)
```

Comment:
Why has this been removed? If it is not required to report the userType in an error scenario then we should change the function definition of errorResult so that it does not have a userType parameter.

https://github.com/hmrc/cgt-property-disposals-frontend/blob/71b83d275d7b7de5d6a3863ae8b0de2c9f5541a2/app/uk/gov/hmrc/cgtpropertydisposalsfrontend/config/ErrorHandler.scala#L38

Reply:
I created an overloaded method which didn't take an `Option[UserType]` and instead took an implicit `RequestWithSessionData[_]` so that we could skip writing `errorHandler.errorResult(request.userType)` everywhere since the user type can be dervied easily from a `RequestWithSessionData[_]`

Reply:
see this: https://github.com/hmrc/cgt-property-disposals-frontend/pull/157/files#diff-8e4beadcc80d3be708ac788c111a3ed4R47

Reply:
don't know if that's going to take you to the right place actually - but it's supposed to take you to the diff for `ErrorHandler.scala`